### PR TITLE
BIP 325: update signet genesis block

### PR DESCRIPTION
**Pull request by ajtowns**
Updates the signet genesis block with higher difficulty and a later timestamp, matching bitcoin/bitcoin#18267 . The new difficulty is the lowest possible such that the target multiplied by two weeks multiplied by 4 (see bitcoind's `CalculateNextWorkRequired()` function) doesn't overflow a uint256.

### CODE REVIEW

File: bip-0325.mediawiki [hunk 1]
```diff
@@ -77,11 +77,12 @@ The genesis block is the same for all signet networks, whereas the message start
...
+The resulting genesis block hash is 00000008819873e925422c1ff0f99f7cc9bbb232af63a077a480a3633bee1ef6, and the block hex is 0100000000000000000000000000000000000000000000000000000000000000000000003ba3edfd7a7b12b27ac72c3e67768f617fc81bc3888a51323a9fb8aa4b1e5e4a008f4d5fae77031e8ad222030101000000010000000000000000000000000000000000000000000000000000000000000000ffffffff4d04ffff001d0104455468652054696d65732030332f4a616e2f32303039204368616e63656c6c6f72206f6e206272696e6b206f66207365636f6e64206261696c6f757420666f722062616e6b73ffffffff0100f2052a01000000434104678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5fac00000000.
```
Comment (jonatack) [inline bip-0325.mediawiki:85]: Checked that these updated values match those in [https://github.com/bitcoin/bitcoin/pull/18267/commits/e8990f1 ](https://github.com/bitcoin/bitcoin/pull/18267/commits/e8990f1) that I have reviewed and manually tested.

**Review by jonatack — COMMENTED** [review]
Did not verify the `CalculateNextWorkRequired` calculation but otherwise LGTM.

**Comment by Yuthana826** [comment]
#1000

**Comment by kallewoof** [comment]
ACK, thanks. Please merge @luke-jr.

**Review by dr-orlovsky — APPROVED** [review]
I confirm that this matches code used in real environment with
- https://github.com/bitcoin/bitcoin/pull/18267#pullrequestreview-491730396
- https://github.com/ElementsProject/lightning/pull/4068
- https://github.com/rust-bitcoin/rust-bitcoin/pull/291
- https://github.com/LNP-BP/docker/blob/master/docker-compose/signet/docker-compose.yml, build of those branches. It works well on my notebook and company server, and bitcoin core, c-lightning and electrs are working in sync.

**Comment by Sjors** [comment]
Maybe update the reference section with more recent mailinglist thread:
* https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-August/018145.html (which points to other resources as well)

**Pull request merged**